components/builder-web/bin/dist

```diff
@@ -10,13 +10,6 @@ mkdir -p dist/assets
 cp -R assets favicon.ico robots.txt opensearch.xml dist
 npm run build
 mkdir -p dist/node_modules
-cp habitat.conf.sample.js dist/habitat.conf.js
 
-css_sha=$(openssl dgst -sha256 assets/app.css | cut -d ' ' -f 2)
-cp assets/app.css "dist/assets/app-${css_sha}.css"
-
-js_sha=$(openssl dgst -sha256 assets/app.js | cut -d ' ' -f 2)
-cp assets/app.js "dist/assets/app-${js_sha}.js"
-sed -e "/id=\"hab-css\"/ s/app\.css/app-${css_sha}\.css/" \
-    -e "/id=\"hab-js\"/ s/app\.js/app-${js_sha}\.js/" \
-    index.html > dist/index.html
```

Comment:
We had been using `openssl` to generate a random string in order to break the browser cache on each release, but all we really need for this is the package's release identifier, which now gets passed in as an argument.

Reply:
👍